command/cp.go

```diff
@@ -429,7 +429,7 @@ func (c Copy) Run(ctx context.Context) error {
 	}
 
 	for object := range objch {
-		if object.Type.IsDir() || errorpkg.IsCancelation(object.Err) {
+		if !object.Type.IsRegular() || object.Type.IsDir() || errorpkg.IsCancelation(object.Err) {
 			continue
```

Comment:
Before these checks getting out of hand what do you think about separating them like;
```suggestion
		if errorpkg.IsCancelation(object.Err) {
			continue
		}

		if !object.Type.IsRegular() || object.Type.IsDir() {
			continue
		}
```